src/app/dashboard/sidebar/sidebar.component.html

```diff
@@ -1,36 +1,44 @@
-<div class="d-flex w-100 justify-content-center">
-  <div>
-    <img routerLink="/landing-screen" src="assets/images/DOE-banner-color.svg" class="doe-logo mx-auto">
+<div class="d-flex justify-content-end sticky-top" (click)="collapseSidebar()">
+  <div class="py-2 px-1 d-flex">
```

Comment:
We need to add css for 'cursor: pointer;' so the user sees they can click on this as a button